src/panels/lovelace/cards/energy/hui-energy-distribution-card.ts

```diff
@@ -1,6 +1,7 @@
 import {
   mdiArrowLeft,
   mdiArrowRight,
+  mdiBattery,
```

Comment:
This is with `mdiCarBattery`:

![image](https://user-images.githubusercontent.com/1444314/128807389-3d059e72-39cc-4c64-b2b5-1e5540340fa9.png)

Maybe more realistic? I like it slightly more.

Reply:
I used `mdiBatteryHigh` in my designs, because `mdiBattery` could also be a vase. `mdiCarBettery` could also work, no strong opinion about this.